tests/test_zk_evm.py

```diff
@@ -1602,7 +1602,7 @@ async def test_execute_at_address(self, zk_evm, eth, params):
             evm_address=evm_contract_address, starknet_address=starknet_contract_address
         ).execute(caller_address=1)
 
-        print("Contract Initiation tx infos")
+        print("Contract initiation tx infos")
```

Comment:
initialization I think? initiation: Initiation is a rite of passage marking entrance or acceptance into a group or society. (wikipedia)

Reply:
Oh yeah haha my bad